torch/csrc/jit/serialization/export.cpp

```diff
@@ -7,6 +7,7 @@
 #include <c10/util/Optional.h>
 #include <c10/util/accumulate.h>
 #include <c10/util/irange.h>
+#include <torch/csrc/api/include/torch/version.h>
```

Comment:
This PR is a good example when seemingly innocuous change causes internal build regressions. Public header should never be included by its full path, but rather by reduced ones, i.e. only include `version.h` as `<torch/version.h>` (as build system is not guaranteed to put it in the same folder as original folder)

Made this change internally with appropriate header order, so clang-format failures for a propose change can be ignored) 
```suggestion
#include <torch/version.h>
```